OneCommunityGlobal/HighestGoodNetworkApp
# Amalesh taking over for Jay - PERMISSION: Toggle Request Bio Permission

# Description
This PR enables the user with the Toggle Request Bio Permission to be able to access and toggle the bio switch.

## Related PRS (if any):
This frontend PR is related to the [#704](https://github.com/OneCommunityGlobal/HGNRest/pull/704) backend PR.

…

## Main changes explained:
- Added a new permission label for toggle request bio.
- Added conditional logic so that the user with the permission can view and toggle the switch.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without Toggle Request Bio under Reports -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to see the toggle on Reports -> Weekly Summaries Report and toggle successfully.

## Screenshots or videos of changes:




https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/326bd5ab-ac0e-4329-969f-88dac8d316af



## Note:
**Unmute the video**
